x-pack/plugins/observability/public/pages/alerts/alerts_flyout/index.tsx

```diff
@@ -90,64 +93,68 @@ export function AlertsFlyout({
     return null;
   }
 
+  const ruleId = alertData.fields['kibana.alert.rule.uuid'] || null;
+  const linkToRule = ruleId && prepend ? prepend(paths.management.ruleDetails(ruleId)) : null;
+
   const overviewListItems = [
     {
-      title: i18n.translate('xpack.observability.alertsFlyout.statusLabel', {
-        defaultMessage: 'Status',
-      }),
+      title: translations.alertsFlyout.statusLabel,
       description: (
         <AlertStatusIndicator
           alertStatus={alertData.active ? ALERT_STATUS_ACTIVE : ALERT_STATUS_RECOVERED}
         />
       ),
     },
     {
-      title: i18n.translate('xpack.observability.alertsFlyout.lastUpdatedLabel', {
-        defaultMessage: 'Last updated',
-      }),
+      title: translations.alertsFlyout.lastUpdatedLabel,
       description: (
         <span title={alertData.start.toString()}>{moment(alertData.start).format(dateFormat)}</span>
       ),
     },
     {
-      title: i18n.translate('xpack.observability.alertsFlyout.durationLabel', {
-        defaultMessage: 'Duration',
-      }),
+      title: translations.alertsFlyout.durationLabel,
       description: asDuration(alertData.fields[ALERT_DURATION], { extended: true }),
     },
     {
-      title: i18n.translate('xpack.observability.alertsFlyout.expectedValueLabel', {
-        defaultMessage: 'Expected value',
-      }),
+      title: translations.alertsFlyout.expectedValueLabel,
       description: alertData.fields[ALERT_EVALUATION_THRESHOLD] ?? '-',
     },
     {
-      title: i18n.translate('xpack.observability.alertsFlyout.actualValueLabel', {
-        defaultMessage: 'Actual value',
-      }),
+      title: translations.alertsFlyout.actualValueLabel,
       description: alertData.fields[ALERT_EVALUATION_VALUE] ?? '-',
     },
     {
-      title: i18n.translate('xpack.observability.alertsFlyout.ruleTypeLabel', {
-        defaultMessage: 'Rule type',
-      }),
+      title: translations.alertsFlyout.ruleTypeLabel,
       description: alertData.fields[ALERT_RULE_CATEGORY] ?? '-',
     },
   ];
 
   return (
     <EuiFlyout onClose={onClose} size="s" data-test-subj="alertsFlyout">
-      <EuiFlyoutHeader>
+      <EuiFlyoutHeader hasBorder>
         <ExperimentalBadge />
         <EuiSpacer size="s" />
         <EuiTitle size="m" data-test-subj="alertsFlyoutTitle">
           <h2>{alertData.fields[ALERT_RULE_NAME]}</h2>
         </EuiTitle>
-        <EuiSpacer size="s" />
-        <EuiText size="s">{alertData.reason}</EuiText>
       </EuiFlyoutHeader>
       <EuiFlyoutBody>
+        <EuiText>
+          <h4>{translations.alertsFlyout.reasonTitle}</h4>
+        </EuiText>
```

Comment:
```suggestion
        <EuiTitle size="xs">
          <h4>{translations.alertsFlyout.reasonTitle}</h4>
        </EuiTitle>
```
I believe we should use `EuiTitle` here https://elastic.github.io/eui/#/display/title